vizro-core/src/vizro/static/js/actions/build_action_loop_callbacks.js

```diff
@@ -76,7 +76,16 @@ export function _gateway(
   return [remaining_actions.slice(1)].concat(trigger_next);
 }
 
-export function _after_action_cycle_breaker(data) {
+function after_action_cycle_breaker(data) {
   document.getElementById("cycle_breaker_div").click();
   return [];
 }
+
+window.dash_clientside = {
```

Comment:
Fantastic! Now, if you put the following code under the `assets/js/custom.js`, it will overwrite the Vizro `collapse_nav_panel` properly and will disable the nav_panel to collapse automatically:

```js
window.dash_clientside.dashboard.collapse_nav_panel = function (n_clicks, is_open) {
    if (!n_clicks) {
      return dash_clientside.no_update;
    }
    if (is_open) {
        return [
          false,
          {
            transform: "rotate(180deg)",
            transition: "transform 0.35s ease-in-out",
          },
          "Show Menu",
        ];
    } else {
      return [
        true,
        {
          transform: "rotate(0deg)",
          transition: "transform 0.35s ease-in-out",
        },
        "Hide Menu",
      ];
    }
}
```